recipe/meta.yaml

```diff
@@ -17,19 +17,19 @@ requirements:
   build:
     - cmake
     - python
-    - numpy 1.8.*
+    - numpy
     - jasper
-    - libpng >=1.6.32,<1.6.35
-    - libnetcdf 4.5.*
-    - hdf5 1.10.1
+    - libpng
+    - libnetcdf
+    - hdf5
     - ecmwf_grib {{ version }}
   run:
     - python
-    - numpy >=1.8
+    - numpy
```

Comment:
Compiling against numpy's C stuff?  If so, you need

``{{ pin_compatible('numpy') }}``